sdcm/remote/kubernetes_cmd_runner.py

```diff
@@ -157,12 +161,14 @@ def __init__(self, kluster, pod_image: str,  # pylint: disable=too-many-argument
         )
 
     def run(self, cmd, **kwargs):  # pylint: disable=arguments-differ
-        if is_scylla_bench_command(cmd) and hasattr(self, "dynamic_remoter") and hasattr(
-                self, "pod_image") and "scylla-bench" not in self.pod_image:
-            LOGGER.info(
-                "Running following 'scylla-bench' command in a separate dynamic loader pod: %s",
-                cmd)
-            return self.dynamic_remoter.run(cmd, **kwargs)
+        if hasattr(self, "dynamic_remoter") and hasattr(self, "pod_image"):
+            if is_scylla_bench_command(cmd) and "scylla-bench" not in self.pod_image:
```

Comment:
The two ends of this if are the same, but diff in log, it can be united, and we should raise exception if it's not any of the supported cmds

Reply:
It doesn't fit our loaders usage needs.
We `must` support not only loader commands but also common ones.
That's is why we check commands to be loader ones and run everything else on the main loader image/container.